include/server/api/route_parameters_grammar.hpp

```diff
@@ -42,6 +42,13 @@ struct RouteParametersGrammar : public BaseParametersGrammar<Iterator, Signature
 
     RouteParametersGrammar(qi::rule<Iterator, Signature> &root_rule_) : BaseGrammar(root_rule_)
     {
+        const auto add_annotation = [](engine::api::RouteParameters &route_parameters,
```

Comment:
due to https://github.com/Project-OSRM/osrm-backend/pull/3628#discussion_r99120126 both members must be updated, so `add_annotation` must remain